packages/experimental/TabList/src/TabList/TabList.tsx

```diff
@@ -36,15 +36,15 @@ export const TabList = compose<TabListType>({
 
       const { disabled, defaultTabbableElement, isCircularNavigation, vertical, ...mergedProps } = mergeProps(tablist.props, final);
 
-      const { animatedIndicatorStyles, canShowAnimatedIndicator, layout, selectedKey } = tablist.state;
+      const { animatedIndicatorStyles, canShowAnimatedIndicator, disabled: disabledState, layout, selectedKey } = tablist.state;
```

Comment:
>disabledState [](http://example.com/codeflow?start=75&length=13)

naming suggestion: include "tablist" in the name? That way when reading disabled on line 47 it's more clear where the disabledState is coming from